adapters/beachfront/beachfront.go

```diff
@@ -111,300 +113,336 @@ type BeachfrontResponseSlot struct {
 	Adm   string  `json:"adm"`
 }
 
-func NewBeachfrontBannerRequest() BeachfrontBannerRequest {
-	r := BeachfrontBannerRequest{}
+func (a *BeachfrontAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var beachfrontRequests beachfrontRequests
+	var errs = make([]error, 0, len(request.Imp))
+	var reqs = make([]*adapters.RequestData, len(beachfrontRequests.Video))
```

Comment:
That sure does make sense, and I moved the reqs = make... line just below preprocess. I got  a run time panic: "runtime error: invalid memory address or nil pointer dereference"

I've moved it back for now, but i'll look in to this once I have the rest cleared up, be cause it sure does seem like it would be an improvement if I can work out where that is happening.